android/src/toga_android/widgets/detailedlist.py

```diff
@@ -4,10 +4,39 @@
 from android.app import AlertDialog
 from android.content import DialogInterface
 from android.graphics import Color, Rect
+from android.os import Build
+from android.util import TypedValue
 from android.view import Gravity, View
 from android.widget import ImageView, LinearLayout, RelativeLayout, ScrollView, TextView
 from java import dynamic_proxy
 
+from .base import Widget
+
+
+def _resolve_theme_color(view, attr_id, fallback):
+    tv = TypedValue()
+    ctx = view.getContext()
+    th = ctx.getTheme()
+    res = ctx.getResources()
+
+    if not th.resolveAttribute(attr_id, tv, True):
+        return fallback  # pragma: no cover
+    if tv.resourceId:
```

Comment:
According to the `resolveAttribute` documentation, when `resolveRefs` is true, as it is here, the TypedValue will never be a resource ID.